connectors/google/google-base/src/main/java/io/camunda/google/model/Authentication.java

```diff
@@ -1,10 +1,10 @@
 /*
  * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
- * under one or more contributor license agreements. Licensed under a proprietary license.
- * See the License.txt file for more information. You may not use this file
- * except in compliance with the proprietary license.
+ *             under one or more contributor license agreements. Licensed under a proprietary license.
+ *             See the License.txt file for more information. You may not use this file
+ *             except in compliance with the proprietary license.
```

Comment:
There's something wrong with your indentation. It's likely in related `pom.xml`

Reply:
fixed